workers/ohsome_quality_analyst/definitions.py

```diff
@@ -86,44 +90,41 @@ def load_metadata(module_name: str) -> Dict:
         A Dict with the class names of the indicators/reports
         as keys and metadata as values.
     """
-    if module_name != "indicators" and module_name != "reports":
-        raise ValueError("module name value can only be 'indicators' or 'reports'.")
-
+    assert module_name == "indicators" or module_name == "reports"
```

Comment:
Because the `module_name` is not something which comes as input from outside. This function is only called twice through our code base and each time the `module_name` is hard coded. For me an assert statement tells me it should never be otherwise. An error on the other hand could be "caught" and used by logging or the API to output information to the user.